core/src/layers/logging.rs

```diff
@@ -546,54 +546,6 @@ impl<A: Accessor> LayeredAccessor for LoggingAccessor<A> {
             .await
     }
 
-    async fn scan(&self, path: &str, args: OpScan) -> Result<(RpScan, Self::Pager)> {
```

Comment:
Will you make a distinction in the `list` side? ditto in other similar places.

Reply:
> Will you make a distinction in the `list` side? ditto in other similar places.

I'm sorry, I didn't understand your comments. Could you please provide more details or clarify what you meant?

Reply:
Since our public API still retains `scan`, do we need to show that this is a `scan` operation and not a `list` anywhere we present it to the user or emphasize this to the user in the documentation that `scan` is an alias of a special `list` operation and it will be displayed as `list`.

Reply:
> do we need to show that this is a `scan` operation and not a `list`

No, scan is an alias of list without delimiter. We will update the docs of scan.